doc/proposals/scorecard-custom-tests-2.md

```diff
@@ -0,0 +1,246 @@
+```
+
+This allows tests to have access to all the bundle contents if they 
+are required for the test logic.  Note:  It is the responsibility of the test to iterate though each CR that they want to test and create Custom Resources as scorecard will no longer be responsible for this.  The list of CRs to test are found in the CSV within the bundle image content.
+
+For the case of executing a test, the bundle image will be passed as a scorecard command flag:
+```
+operator-sdk scorecard --bundle='quay.io/myorg/mytest' --selector=suite=basic
+```
+
+#### Test Output
+
+Custom tests are expected to produce log output that conforms to the
+scorecard v1alpha2 Test Result output.  This allows scorecard to parse
+the custom test log output and aggregate the results like any other
+scorecard test.
+
+The scorecard expected output would be in JSON, and would match the following definition:  https://github.com/operator-framework/operator-sdk/blob/master/pkg/apis/scorecard/v1alpha2/types.go#L36
+
+The scorecard v1alpha2 Test Result output contains information such as
+pass or fail status and suggestions/errors produced by the test that
+the end user would want to see in the aggregated scorecard results.
+
+
+### Custom Test Development Model
+
+The developer of a custom test would have a workflow similar to:
+ 
+ * write a custom test binary in the language of their choice, sample custom test within the SDK repository would be available as a starting point
+ * test the custom test binary locally (out-of-cluster) to make sure the output is produced that matches v1alpha2 ScorecardTestResult format
+ * create a scorecard DSL yaml file to run their custom test image
+ * when the test binary works as expected, the developer would build and push the test image to a container registry accessible to the scorecard test environment
+ * the developer would rebuild their bundle image passing in their scorecard DSL file
+ * the Basic and OLM test manifests would be added automatically to the bundle image as part of the bundle image build process.  This allows users to run the Basic and OLM tests along with their custom tests.
+ * lastly, the developer would run the scorecard passing in their operator bundle image they just built along with any other scorecard flags (e.g. selector).
+
+### Custom Test Packaging and Configuration
+
+In this design proposal, custom tests are container images, described in the scorecard DSL yaml file.  End users would add their scorecard DSL yaml file into their operator bundle image or pass it into scorecard using a command line flag.  Custom test images are stored in a container registry outside the control of scorecard.
+
+### Scorecard Configuration
+
+The scorecard DSL yaml file would be accessible from a local disk location to scorecard or from within a bundle image. 
+
+Scorecard would assume that the scorecard DSL yaml file would be located in the bundle image at the following location:
+```
+/scorecard/.osdk-scorecard.yaml
```

Comment:
Let's take an opportunity to rename this to a non-hidden file.

How about simply `/scorecard/config.yaml`?

Reply:
updated in latest commit.